packages/apps/spaces/components/organization/organization-list/OrganizationList.tsx

```diff
@@ -230,7 +235,7 @@ export const OrganizationList: React.FC<OrganizationListProps> = ({
 
       {/* TODO: Remove coercion to any type when we get rid of the old graphql types generated which are out of sync */}
       <Table<Organization>
-        data={(data as Organization[]) ?? []}
+        data={(data as unknown as Organization[]) ?? []}
```

Comment:
Why unknown is needed here? it was working with previous, direct  type assercion 🤔

Reply:
build fails because of this for no reason unless unknown is casted first. There a failed build which can be found in the actions that shows this error.